webapi/webapi-webgl-khronos-tests/webgl/khronos/resources/js-test-pre.js

```diff
@@ -140,6 +168,14 @@ function escapeHTML(text)
 
 function testPassed(msg)
 {
+    if (msg !== "successfullyParsed is true") {
```

Comment:
We don't need to count the sub-case numbers; we only need to record a pass or fail for the whole test page. Does this change indicate this pass/fail criteria?